freud/locality.pyx

```diff
@@ -482,94 +462,48 @@ cdef class NeighborList:
 
     def __getitem__(self, key):
         R"""Access the bond array by index or slice."""
-        cdef size_t n_bonds = self.thisptr.getNumBonds()
-        cdef size_t[:, ::1] neighbors
-        if not n_bonds:
-            result = np.empty(shape=(0, 2), dtype=np.uint64)
-        else:
-            neighbors = <size_t[:n_bonds, :2]> self.thisptr.getNeighbors()
-            result = np.asarray(neighbors[:, :], dtype=np.uint64)
-        result.flags.writeable = False
-        return result[key]
+        return freud.util.make_managed_numpy_array(
+            &self.thisptr.getNeighbors(),
+            freud.util.arr_type_t.UNSIGNED_INT)[key]
 
     @property
-    def index_i(self):
+    def query_point_index(self):
```

Comment:
Should we also make this plural for correctness? `query_point_indices` seems better (and `point_indices` below).

Reply:
Answering my own question: yes, we should. The other attributes are all plural too.